### PR TITLE
Add git how-to

### PR DESCRIPTION
**Pull request by khdc-me**

### CODE REVIEW

**Comment by ovv** [comment]
Looks good for me :+1: 

Thanks

**Review by ovv — DISMISSED** [review]

File: how-to-git.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+Keeping your forked/master in tact allows others to more easily work with your forked copy.
+
+Here are the steps with links to documentation:
+
+1. [Fork to your GitHub account](https://help.github.com/articles/fork-a-repo/#fork-an-example-repository)
+1. [Clone your forked copy onto your computer](https://help.github.com/articles/fork-a-repo/#step-2-create-a-local-clone-of-your-fork)
+1. [Set master project repo as the upstream for your cloned repo](https://help.github.com/articles/configuring-a-remote-for-a-fork/)
+1. Create a new branch **(don't develop on 'Master')**
+    > git checkout -b <new-branch-name>
```
Comment (mattrasband) [inline how-to-git.md]: Formatted this makes it a:

> quote

Vs maybe preferring 

```bash
$ echo "The shell"
```

Either probably works fine, but figured it is worth pointing out

**Review by mattrasband — DISMISSED** [review]
Minor comment about format style, not a big deal and primarily _preference_ for syntax highlighting.

**Comment by khdc-me** [comment]
Thanks for pointing that out Matt. I've updated it.

File: how-to-git.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+Keeping your forked/master in tact allows others to more easily work with your forked copy.
+
+Here are the steps with links to documentation:
+
+1. [Fork to your GitHub account](https://help.github.com/articles/fork-a-repo/#fork-an-example-repository)
+1. [Clone your forked copy onto your computer](https://help.github.com/articles/fork-a-repo/#step-2-create-a-local-clone-of-your-fork)
+1. [Set master project repo as the upstream for your cloned repo](https://help.github.com/articles/configuring-a-remote-for-a-fork/)
+1. Create a new branch **(don't develop on 'Master')**
+    ```
```
Comment (mattrasband) [inline how-to-git.md]: Sorry I omitted this before, but you can put the syntax name (in this case "bash") after the ``` for highlighting, compare:

```
$ echo "without the bash name behind the \`\`\`"
```

```bash
$ echo "with the bash name behind the \`\`\`"
```

**Review by mattrasband — APPROVED** [review]

**Comment by mattrasband** [comment]
Sorry for the runaround on the highlighting, it didn't end up doing a ton in the end :'(

**Comment by khdc-me** [comment]
No prob. I prefer addressing those types of details (might as well do it right). Not sure why the highlighting didn't work as I thought it would, but will be careful to include it from the start next time.

**Pull request merged**